velox/exec/tests/utils/Cursor.cpp

```diff
@@ -113,7 +113,9 @@ TaskCursor::TaskCursor(const CursorParameters& params)
   if (params.queryCtx) {
     queryCtx = params.queryCtx;
   } else {
-    queryCtx = core::QueryCtx::createForTest();
+    executor_ = std::make_shared<folly::CPUThreadPoolExecutor>(
```

Comment:
Like above, this is not sufficient. I think Cursor should not make its own executor, but rather require an executor to be passed in.